Apps/frontend/src/views/Indicators.svelte

```diff
@@ -7,35 +7,61 @@
     INDICATOR_SECTION_WIDTH,
     INDICATOR_WIDTH,
     INDICATOR_ZERO_LINE_COLOR,
-    LINE_COLORS_RGBA,
-    NUM_INTERVALS_HORIZONTAL,
+    LINE_COLORS_RGBA, NUM_CHANNELS,
+    NUM_INTERVALS_HORIZONTAL
   } from "../const";
   import { roundVoltage } from "../helper";
 
   let canvasElement;
   let canvasContext;
-  let min = Array(10).fill(0.0);
-  let max = Array(10).fill(0.0);
+  let current = Array(NUM_CHANNELS).fill(0.0);
+  let min = Array(NUM_CHANNELS).fill(0.0);
+  let max = Array(NUM_CHANNELS).fill(0.0);
+  let offsets = Array(NUM_CHANNELS).fill(0.0);
+  let scalings = Array(NUM_CHANNELS).fill(1.0);
+  let startStopLine = Array(NUM_CHANNELS).fill(true);
```

Comment:
Wasn't quite sure what you ment with this variable, guess it is more like `isChannelRunning` or `isChannelStarted`/`hasChannelStarted`?

Reply:
Good point, I'm not happy with the naming, too. But I stole it from the other components to have a consistent naming and change it in the next refactoring :D

Reply:
Regarding the bracket, I don't think this is in scope of the issue.
But I think that would be a question for monday.

Reply:
I added a snapshot testing module to cypress and one test testing the initial state of the indicators, however:
- I'm getting an error in cypress.config.js when importing the plugin without the .js extension, but without the extension cypress throws an error. Don't know why...
- For testing the functionality of offsets and scalings we need some way to mock input on the controls. For that we either need to mock the generator to send a static signal and simulate the controls with cy.click(), which would make the tests dependent on the generator. Or we control the indicators programmatically (better), but this is not possible with cypress from what I learned...

What's your opinion on that?

Reply:
Sounds like a sticky situation. Sadly I'm not all that good with Cypress, maybe @jenswaechtler knows how to best approach this issue.